lib/stacker/stack/parameters.rb

```diff
@@ -18,9 +18,17 @@ def region_defaults
         stack.region.defaults.fetch 'parameters', {}
       end
 
+      def global_defaults
+        stack.region.defaults.fetch 'globalParameterFile', {}
```

Comment:
nit: i'd probably just name it `parametersFile` or something. it's only "global" if all the regions refer to the same file, but that's determined by particular usage.

can you comment on the priority of all the settings? i think we'll use the *first* available of:

1. stack parameter
2. region default parameter
3. region default parameter file
4. template default

is that right?

Reply:
yea that's a good point. I'll change the name
and yes that's correct about the priority. do you want me to add a comment in the stacker code or just in this pr?

Reply:
made the changes and added more detail to the comment in stacker. can you take another look @jfhamlin ?
also tested these changes in the infra repo on my branch